schrack/cion-registers.json

```diff
@@ -0,0 +1,124 @@
+{
+    "protocol": "RTU",
+    "endianness": "BigEndian",
+    "blocks": [
+        {
+            "id": "e3",
+            "readSchedule": "update",
+            "registers": [
+                {
+                    "id": "currentChargingCurrentE3",
+                    "address": 126,
+                    "size": 1,
+                    "type": "uint16",
+                    "registerType": "holdingRegister",
+                    "readSchedule": "update",
+                    "description": "Current charging Ampere",
+                    "defaultValue": 6,
+                    "access": "R"
+                },
+                {
+                    "id": "maxChargingCurrentE3",
+                    "address": 127,
+                    "size": 1,
+                    "type": "uint16",
+                    "registerType": "holdingRegister",
+                    "readSchedule": "update",
+                    "description": "Maximum charging current of connected cable",
+                    "defaultValue": 32,
+                    "access": "R"
+                }
+            ]
+        }
+    ],
+    "registers": [
+        {
+            "id": "chargingEnabled",
+            "address": 100,
+            "size": 1,
+            "type": "uint16",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "Charging enabled",
+            "defaultValue": 0,
+            "access": "RW"
+        },
+        {
+            "id": "chargingCurrentSetpoint",
+            "address": 101,
+            "size": 1,
+            "type": "uint16",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "Allowed charging current",
+            "defaultValue": 6,
+            "access": "RW"
+        },
+        {
+            "id": "statusBits",
+            "address": 121,
+            "size": 1,
+            "type": "uint16",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "Maximum charging current",
+            "defaultValue": 0,
+            "access": "R"
+        },
+        {
+            "id": "pluggedInDuration",
+            "address": 153,
+            "size": 2,
+            "type": "uint32",
+            "registerType": "holdingRegister",
+            "readSchedule": "update",
+            "description": "Plugged in duration",
+            "defaultValue": 6,
```

Comment:
I find it useful to set also the unit as string coming from the register, so when you look the next time to the code or you are about to use this property you know right away which unit the register provides from the comments.

Reply:
done